components/WotwSeedgen/PresetButton.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+    <v-row>
+      <v-btn @click='$emit("load-preset", presetName)'>{{ presetName }}
+      </v-btn>
```

Comment:
```suggestion
      <v-btn @click='$emit("load-preset", presetName)'>{{ presetName }}</v-btn>
```

This actually matters because it inserts a space at the end.